File: demo/models/StandingInstruction.py
from django.db import models
from demo.models.StandingInstructionFrequency import StandingInstructionFrequency
from demo.models.StandingInstructionStatus import StandingInstructionStatus

#======================================================================
# 
# Encapsulates data for model StandingInstruction
#
# @author your_name_here
#
#======================================================================

#======================================================================
# Class StandingInstruction Declaration
#======================================================================
class StandingInstruction (models.Model):

#======================================================================
# attribute declarations
#======================================================================
	instructionId = models.CharField(max_length=200, null=True)
	amount = Money
	nextExecutionDate = models.DateField(null=True)
	account = models.ForeignKey('Account', on_delete=models.CASCADE, null=True, blank=True, related_name='+')
	beneficiary = models.ForeignKey('ExternalAccount', on_delete=models.CASCADE, null=True, blank=True, related_name='+')
	frequency = models.CharField(max_length=64, null=True, choices=[(tag.name, tag.value) for tag in StandingInstructionFrequency])
	status = models.CharField(max_length=64, null=True, choices=[(tag.name, tag.value) for tag in StandingInstructionStatus])

#======================================================================
# function declarations
#======================================================================
	def toString(self):
		str = ""
		str = str + self.instructionId
		str = str + self.amount
		str = str + self.nextExecutionDate
		str = str + self.frequency
		str = str + self.status
		return str;
    
	def __str__(self):
		return self.toString();

	def identity(self):
		return "StandingInstruction";
    
	def objectType(self):
		return "StandingInstruction";

File: bankingonaspdotnet/Domain/StandingInstruction.cs
using bankingonaspdotnet.Contracts;

namespace bankingonaspdotnet.Domain;

public class StandingInstruction
{
    public Guid Id { get; set; } = Guid.NewGuid();

 public virtual long? StandinginstructionId { get; set; } 
 public virtual string? InstructionId { get; set; } 
 public virtual Money? Amount { get; set; } 
 public virtual DateOnly? NextExecutionDate { get; set; } 
public virtual Account? Account { get; set; } 
public virtual ExternalAccount? Beneficiary { get; set; } 
 public virtual StandingInstructionFrequency? Frequency { get; set; } 
 public virtual StandingInstructionStatus? Status { get; set; } 

    public static StandingInstruction FromRequest(StandingInstructionRequest request) {
        return new StandingInstruction {
            Id = request.Id,
            InstructionId = request.InstructionId,
            Amount = request.Amount,
            NextExecutionDate = request.NextExecutionDate,
            Frequency = request.Frequency,
            Status = request.Status,
        };
    }
}
